neo4j/internal/bolt/connect.go

```diff
@@ -35,12 +37,18 @@ type protocolVersion struct {
 	back  byte // Number of minor versions back
 }
 
-// Supported versions in priority order
+func (p *protocolVersion) formatProtocol() string {
+	return fmt.Sprintf("0x%04X%02X%02X", p.back, p.minor, p.major)
```

Comment:
```suggestion
	return fmt.Sprintf("%#04X%02X%02X", p.back, p.minor, p.major)
```
For consistent formatting (`0X0000...` instead of `0x0000...`).